app/about/about.html

```diff
@@ -33,7 +33,7 @@ <h2 class="md-toolbar-tools">
         <md-card>
             <md-card-content>
             <a href="https://developers.google.com/">
-                <img class="sponsor" src="/app/images/Google-Developers.png" alt="Google Developers"
+                <img class="sponsor" src="/app/images/logo_lockup_developers_icon_vertical.png" alt="Google Developers"
```

Comment:
I know it's nitpick (so forgive me) but could you rename the file to make it a bit easier to recognize it as the Google Developers logo? Or is there a benefit of using "Google's" naming?